GUIDE.md

```diff
@@ -1,50 +1,50 @@
-# Компоненты YARCH
+# YARCH components
 
 ## Builder
-Отвечает за сборку модуля. В нем создаются необходимые сущности и проставляются зависимости.
+Responsible for the assembly of the module. It creates the necessary entities and shows dependencies.
```

Comment:
Builder is responsible for the assembly of the module. It creates all necessary entities and shows dependencies.